llvm/test/Analysis/CostModel/AArch64/arith-fp-sve-frem.ll

```diff
@@ -0,0 +1,98 @@
+; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py UTC_ARGS: --filter "cost.*vscale.*frem" --version 4
+
+; RUN: opt -mattr=+sve -passes=loop-vectorize -debug-only=loop-vectorize -disable-output -S < %s 2>&1 | FileCheck %s --check-prefix=NO-VECLIB
```

Comment:
could you also addd RUN lines for Neon, i.e without +sve? That wilol require changing the file name as well.